guides/cookbook.md

```diff
@@ -1632,6 +1906,55 @@ class CsvToIon
 ```
 </div>
 
+<div class="tabpane Go" markdown="1">
+
+```Go
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	
+	"github.com/amzn/ion-go/ion"
+)
+
+func main() {
+	file, er := os.Open("c://values.cvs")
```

Comment:
Where does the CVS file come from?

Reply:
It supposed to be in the directory of where this test is going to be tested. There is explanation about the file in the cookbook and even examples in other languages use the file just like this.